docs/xshareclasses.md

```diff
@@ -264,6 +264,8 @@ case, the VM continues without using shared classes.
 
 : On AIX, Linux, and macOS systems, if a user creates a cache by specifying the `groupAccess` suboption, other users in the same group must also specify this suboption to be granted access to the same cache.
 
+: It is recommended to use [`cacheDir`](xshareclasses.md#cacheDir) to set a location for the cache. If necessary, use [`cacheDirPerm`](xshareclasses.md#cacheDirPerm) to ensure that the cache directory permissions are set appropriately. The default directory for a cache is `/tmp/javasharedresources`. Some systems may clear the contents of the `/tmp` directory on a reboot, removing the shared cache. 
```

Comment:
> It is recommended to use `cacheDir`
> If necessary, use `cacheDirPerm`

The options are `cacheDir=` and `cacheDirPerm=`

> The default directory for a cache is `/tmp/javasharedresources`

Even though this is under the "groupAccess" section, I feel it is better to mention groupAccess in this sentence. So better to say: "The default directory for a shared cache is `/tmp/javasharedresources` if sub-option `groupAccess` is specified"

Reply:
> The options are cacheDir= and cacheDirPerm=

The user can see that when they follow the links. In the listing they are `cacheDir` and `cacheDirPerm`. This is consistent with other places in the doc that refer to sub-options.

Addressed the second point.